internal/pkg/agent/install/install_test.go

```diff
@@ -185,3 +187,10 @@ func TestCopyFiles(t *testing.T) {
 	}
 
 }
+
+func TestInstallMarker(t *testing.T) {
```

Comment:
Maybe add some assertion about ownership and permissions?
I know it's not vital as we install/uninstall as root/administrator but it's part of the install marker creation and it would be nice to see this part of `CreateInstallMarker` behavior documented in the test as well